plugins/regex_remap/regex_remap.cc

```diff
@@ -400,11 +400,14 @@ RemapRegex::compile(const char *&error, int &erroffset)
     return -1;
   }
 
-  _extra = pcre_study(_rex, 0, &error);
-  if ((_extra == nullptr) && (error != nullptr)) {
+  _extra = pcre_study(_rex, PCRE_STUDY_EXTRA_NEEDED, &error);
+  if (error != nullptr) {
```

Comment:
Should you still have a test somewhere to return if _extra is nullptr before you dereference it?

Reply:
No. Because of the `PCRE_STUDY_EXTRA_NEEDED`, the returned value will only be `nullptr` if memory is exhausted, in which case we're already over the cliff.